lib/report_formatter/timeline.rb

```diff
@@ -106,12 +106,21 @@ def tl_event(tl_xml, row, col)
             ems = ExtManagementSystem.find(rec[:ems_id])
             ems_cloud =  true if ems.kind_of?(EmsCloud)
           end
-          if rec[:vm_name] && !ems_cloud             # Create the title using VM name
-            e_title = rec[:vm_name]
-          elsif rec[:host_name] && !ems_cloud                 #   or Host Name
-            e_title = rec[:host_name]
-          elsif rec[:ems_cluster_name] && !ems_cloud          #   or Cluster Name
-            e_title = rec[:ems_cluster_name]
+          if !ems_cloud
```

Comment:
@dkorn I expect to have a:

    when "Container"

what does it take for that? (I assume we're missing something if you're not doing it)